podman_compose.py

```diff
@@ -1032,12 +1029,12 @@ async def container_to_args(compose, cnt, detached=True):
                 podman_args.append("--no-healthcheck")
             elif healthcheck_type == "CMD":
                 cmd_q = "' '".join([cmd_quote(i) for i in healthcheck_test])
-                podman_args.extend(["--healthcheck-command", "/bin/sh -c " + cmd_q])
+                podman_args.extend(["--health-cmd", "/bin/sh -c " + cmd_q])
```

Comment:
I don't think in `CMD` case this should be executed with ` /bin/sh` prevents cases from working where there is no  `/bin/sh` in the image like minimal images for Rust applications. I don't have docker to verify, but from the documentation, my understanding is that this is ran without shell.